test/src/specs/transaction/depend_tx_in_same_block.rs

```diff
@@ -0,0 +1,57 @@
+use crate::{sleep, Net, Spec};
+use log::info;
+
+pub struct DepentTxInSameBlock;
```

Comment:
Please move this spec to specs/mining.rs

Reply:
I'm not sure, even this bug caused by miner, in general, this is a transaction/block verifier related spec.

Reply:
Verifier uses different logic to verify cellbase capacity, this PR only fix the bug in miner's `create_cellbase_transaction` function, so it's better to move this spec to `mining.rs`